src/shared/theme.js

```diff
@@ -1,10 +1,105 @@
 'use strict';
 
+var _ = require('lodash');
```

Comment:
Let's try to stay away from as many unnecessary 3rd party dependencies as possible, and stick to native JavaScript methods and patterns as much as possible. I don't think lodash is necessary here.